src/components/hooks/useValidation.js

```diff
@@ -0,0 +1,61 @@
+import { useCallback, useState } from 'react';
+
+/**
+ * Form validation hook.
+ * Returns an object with the following fields:
+ * - values {name: string, email: string, password: string} with form values set by user
+ * - isValid {boolean} when true, the form is valid
+ * - handleChange {(e: ChangeEvent) => void} change handler. You must add it to the form being validated
+ * - resetForm {() => void} reset handler
+ */
+
+export default function useValidation() {
```

Comment:
между jsdoc и тем, что он документирует не должно быть пустых строк
```suggestion
 * - resetForm {() => void} reset handler
 */
export default function useValidation() {
```